### PR TITLE
DE29525-fire custom event when remove banner button is clicked

### PR DESCRIPTION
**Pull request by edwardg8**

### CODE REVIEW

File: d2l-image-banner-overlay.html [hunk 1]
```diff
@@ -413,6 +413,11 @@ <h1 id="courseName">[[_courseName]]</h1>
 					return;
 				}
 
+				this.dispatchEvent(new CustomEvent(
+					'd2l-image-banner-overlay-remove-banner',
```
Comment (trevorjonesd2l) [inline d2l-image-banner-overlay.html]: I believe this would get called on both an `add` and `remove` of the banner

Comment (edwardg8) [reply d2l-image-banner-overlay.html]: yep, I think that's the behavior we want, remove is from the banner "Remove this banner" button, and add when user clicks "Undo", both need partial render the homepage menu

Comment (njostonehouse) [reply d2l-image-banner-overlay.html]: Maybe calling it `d2l-image-banner-overlay-toggle` would be more accurate?

Comment (trevorjonesd2l) [reply d2l-image-banner-overlay.html]: Should the event be named something like `'d2l-image-banner-overlay-toggle-banner'` in that case?

**Pull request merged**